Translation Program page revamp (#7289)
ethereum/ethereum-org-website: src/content/contributing/translation-program/resources/index.md

---
title: Resources for translators
lang: en
description: Useful resources for ethereum.org translators
sidebar: true
---

# Resources {#resources}

You can find some useful guides and tools for ethereum.org translators, as well as translation communities and updates below.

## Guides {#guides}

> - [Translation Style Guide](/contributing/translation-program/translators-guide/) _– instructions and tips for ethereum.org translators_

Reviewer: ```suggestion
- [Translation style guide](/contributing/translation-program/translators-guide/) _– instructions and tips for ethereum.org translators_
```

- [Translation FAQs](/contributing/translation-program/faq/) _– frequently asked questions and answers about the ethereum.org Translation Program_
- [Crowdin online editor guide](https://support.crowdin.com/online-editor/) _– an in-depth guide to using the Crowdin online editor and some of Crowdin's advanced features_
- [Content buckets](/contributing/translation-program/content-buckets/) _– which pages are included in each content bucket of ethereum.org_

## Tools {#tools}

- [Microsoft Language Portal](https://www.microsoft.com/en-us/language)
  _– useful for finding and checking the standard translations of technical terms_
- [Linguee](https://www.linguee.com/)
  _– search engine for translations and dictionary that enables searching by word or phrase_
- [Proz term search](https://www.proz.com/search/)
  _– database of translation dictionaries and glossaries for specialized terms_
- [Eurotermbank](https://www.eurotermbank.com/)
  _– collections of European terminology in 42 languages_

## Communities {#communities}

> - [Language-specific Discord translation groups](https://discord.gg/6WX7E97)

Reviewer: We should get in the habit of using our /discord/ link. 

```suggestion
- [Language-specific Discord translation groups](/discord/)
```

Reviewer: @konopkja FYI we can track Discord link usage from our server by creating unique links for pages we want to track, but I assume Matomo provides us with this functionality already?

Reviewer 2: matomo tracks external links usage so we can filter out and see clicks on a specific link

>   _– an initiative to connect ethereum.org translators into Translation Groups_

Reviewer: ```suggestion
  _– an initiative to connect ethereum.org translators to Translation Groups_
```

- [Chinese translators group](https://www.notion.so/Ethereum-org-05375fe0a94c4214acaf90f42ba40171)
  _– Notion page for easier coordination between Chinese translators_

## Latest updates {#latest-updates}

To keep up-to-date with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/):

- [October 2021 milestones update](https://blog.ethereum.org/2021/10/04/translation-program-update/)
- [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
- [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
- [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)

## Office hours for translators {#office-hours}

We have office hours for translators on the second Wednesday of every month. These are held in the #office-hours voice channel on the [ethereum.org Discord](https://discord.gg/geKhWjtF), where you can also find the exact times and additional details.

> The purpose of office hours is to allow our translators to ask questions about the translation process, provide feedback on the program, share their ideas, or just have a chat with us.

Reviewer: ```suggestion
Office hours allow our translators to ask questions about the translation process, provide feedback on the program, share their ideas, or just chat with the core ethereum.org team.
```

> Participants can also learn more about the Translation Program and get some information on becoming an ethereum.org translator.

Reviewer: ```suggestion
```

> Participants can also learn more about the Translation Program and get some information on becoming an ethereum.org translator.

Reviewer: This seems to be covered in the previous sentence
